pkg/kudoctl/kudoinit/setup/wait.go

```diff
@@ -38,31 +38,14 @@ func verifyKudoDeployment(client corev1.PodsGetter, namespace string) (bool, err
 // isKUDOPodReady fetches the KUDO pod running in the given namespace and returns true if Ready Condition has a status of true
 func isKUDOPodReady(client corev1.PodsGetter, namespace string) (bool, error) {
 	selector := manager.GenerateLabels().AsSelector()
-	pod, err := getFirstRunningPod(client, namespace, selector)
-	if err != nil {
-		return false, err
-	}
-
-	s, err := managerContainerStatus(pod)
-	if err != nil {
+	pod, err := getFirstReadyPod(client, namespace, selector)
```

Comment:
Now that I'm looking at it again: why do we check the individual manager pods? It is a `StatefulSet` so why not simply check that with the healthutil?

Reply:
I thought replica does not have enough information about readiness but it does so ... thank you :) you're right